platforms/corda-ent/charts/signer/values.yaml

```diff
@@ -112,14 +112,14 @@ idmanPort:
 serviceLocations:
  # Provide the idman service address
   identityManager:
-    # Ex. host: idman-internal
+    # Ex. host: idman.namespace
```

Comment:
I think this is to be internal. Please check idman and nmap charts about the service name.

Reply:
the corda cenm charts do have service names as idam-internal or nmap-internal, but its upto our implementation, for consistency we have removed internal from all service names